playbooks/scenario1_5_setup.yml

```diff
@@ -1,11 +1,46 @@
 ---
-- name: Register existing settings
+- name: Build infrastructure
   hosts: local
+  become: yes
+  roles:
+    - build_infrastructure
+  vars:
+    terraform_base_dir: "../scenarios"
+    scenario_name: "scenario1_5"
+    key_name: "key_scenario1_5"
+    requirements: "requirements.txt"
+    infra_state: present
+
+- name: Setting terraform outputs
+  hosts: local
+  become: yes
+  vars:
+    bastion_public_ip: "{{ tf_output.outputs[\"out-scn1_5_lb_fip\"].value }}"
```

Comment:
Bastion public IP = out-scn1_5_lb_fip
Loadbalancer public IP = out-scn1_5_bastion_fip
Is it ok?

Reply:
tf_output.outputs[\"out-scn1_5_lb_fip\"].value gives  value out-scn1_5_lb_fip of dictionary  tf_output.outputs

Reply:
i am sorry,
 you are right